docs/89_glossary.md

```diff
@@ -402,7 +504,7 @@ reported tabulation outcome.
 
 * **county number** as part of the ballot manifest
```

Comment:
We seem to be using county names in the manifest, following the original CDOS example.